### DISCUSSION TITLE
21-06-21 일일회고

### DISCUSSION THREAD

**Discussion by GleamingStar** [d-h-k/issue-tracker#85] [회고]
오늘 수업시간중 크롱이 다녀갔습니다;;
그래서 좀 일찍 올립니다

**Comment by GleamingStar** [comment]
크롱 예시 #80

**Comment by GleamingStar** [comment]
오늘은 개발을 1줄도 못했네요... 저녁에 몇줄 더 커밋해보도록 하겠습니다!

**Comment by eeeesong** [comment]
아이고 어제는 까맣게 잊었네요
뒤늦게 적어보자면.. 
모의면접 수업을 무사히?듣고, 잭슨과 함께 2주차 PR을 보냈습니다!
PR 보내기 전 코드 정리에 시간이 꽤 들어서 이후엔 이번주 계획을 짠 뒤, 이슈 작성 화면의 상단 메뉴 정도만 만들고 하루를 끝낸 것 같네요!
수요일까지 기능 개발은 마무리하고, 목요일엔 테스트 코드를 작성해보기로 했습니다.

**Comment by JacksonPk** [comment]
매번 까먹지 않는다고 하구 까먹네요 ㅜㅜ 이번주는 이슈에 대해서 필수적으로 중요한 것만 구현 하려구 합니다.
마지막까지 다들 화이팅입니다.
